kodilla-stream/src/main/java/com/kodilla/stream/forumuser/ForumUser.java

```diff
@@ -0,0 +1,48 @@
+package com.kodilla.stream.forumuser;
+
+import java.time.LocalDate;
+
+public final class ForumUser {
+
+    private final int userId;
+    private final String userName;
+    private final char sex;
+    private final LocalDate birthDate;
+    private final int postsNumber;
+
+    public ForumUser(final int userId, final String userName, final char sex, final int yearOfBirth,
+                     final int monthOfBirth, final int dayOfBirth, final int postsNumber) {
+        this.userId = userId;
+        this.userName = userName;
+        this.sex = sex;
+        this.birthDate = LocalDate.of(yearOfBirth, monthOfBirth, dayOfBirth);
+        this.postsNumber = postsNumber;
+    }
+
+    public int getUserId() {
+        return userId;
+    }
+
+    public char getSex() {
+        return sex;
+    }
+
+    public LocalDate getBirthDate() {
+        return birthDate;
+    }
+
+    public int getPostsNumber() {
+        return postsNumber;
+    }
+
+    @Override
+    public String toString() {
```

Comment:
Jeśli chodzi o metodę `toString()` to oczywiście można użyć wygenerowanej przez Ideę metody, ale fajnie też nauczyć się czegoś nowego, prawda? Często używana biblioteką jest `gson`.
Kod wygląda wówczas bardzo prosto i w każdej klasie tak samo :)
`publis String toString() { return new GsonBuilder().setPrettyPrinting().create().toJson(this); }`
Maven dependency:
<dependency>
    <groupId>com.google.code.gson</groupId>
    <artifactId>gson</artifactId>
    <version>2.8.5</version>
</dependency>